controller/main_controller.php

```diff
@@ -93,7 +93,7 @@ public function display($route)
 	* @param string $route The route name for a page
 	* @return object $entity The entity object
 	* @throws http_exception
-	* @access public
+	* @access protected
```

Comment:
In the core it is still used sometimes yes (mostly when the visibility isn't specified) but it should be changed yes.